game-app/game-headed/src/main/java/games/strategy/engine/framework/startup/ui/PlayerSelectorRow.java

```diff
@@ -59,23 +58,19 @@ public class PlayerSelectorRow implements PlayerCountrySelection {
     this.playerTypesProvider = playerTypes;
 
     enabledCheckBox = new JCheckBox();
-    final ActionListener disablePlayerActionListener =
-        new ActionListener() {
-          @Override
-          public void actionPerformed(final ActionEvent e) {
-            if (enabledCheckBox.isSelected()) {
-              enabled = true;
-              // the 1st in the list should be human
-              PlayerSelectorRow.this.playerTypes.setSelectedItem(HeadedPlayerTypes.HUMAN_PLAYER);
-            } else {
-              enabled = false;
-              // the 2nd in the list should be Weak AI
-              PlayerSelectorRow.this.playerTypes.setSelectedItem(PlayerTypes.WEAK_AI);
-            }
-            setWidgetActivation();
+    enabledCheckBox.addActionListener( // disablePlayerActionListener
```

Comment:
nit: the ` // disablePlayerActionListener` might be a stray thing that can be deleted I think.